dds_web/api/project.py

```diff
@@ -145,7 +145,10 @@ def post(self):
 
             try:
                 project.project_statuses.append(new_status_row)
-                project.busy = False
+                flask.current_app.logger.info(
+                    f"Setting busy status. Project: '{project.public_id}', Busy: False"
+                )
+                project.busy = False # TODO: Use set_busy instead? 
                 db.session.commit()
```

Comment:
The logging in `set_busy` is before the commit since it only indicates what is about to happen. But if we want to move the logging here to after the commit we can change the message to `Busy status set. Project: '{project.public_id}', Busy: False`. I agree that it might be better.